enum LoginOpcode
{
	ID_C2S_PLAYER_LOGIN_REQUEST			= 0x00010001;	//玩家登陆请求
	ID_S2C_PLAYER_LOGIN_RESPONSE		= 0x00010002;	//玩家登陆响应
}

//ID_C2S_PLAYER_LOGIN_REQUEST 
message CMsgPlayerLoginRequest
{
	required string username = 1;
	required string password = 2;
}

//ID_S2C_PLAYER_LOGIN_RESPONSE 
message CMsgPlayerLoginResponse
{
	required string entityId = 1;
}